core/src/main/kotlin/configuration.kt

```diff
@@ -10,6 +10,13 @@ enum class Platform(val key: String) {
     native("native"),
     common("common");
 
+    fun toPrettyString() = when(this){
+            jvm -> "JVM"
+            js -> "JS"
+            common -> "Common"
+            native -> "Native"
+    }
```

Comment:
Unneeded, `Platform` should only be used to identify passes and to configure analysis properly